behaviours/__init__.py

```diff
@@ -43,7 +43,11 @@
 ]
 
 _reaction_add_chain: _Chain[hikari.GuildReactionAddEvent] = [
-    [userinfo.analyse_reaction, meme_rater.respond_to_question_mark]
+    [
```

Comment:
As it stands this means the 💩/❓ get added to a user's total emoji stats, is that what we want?

Reply:
That's not what we want. How to prevent this?

Reply:
exclude those 2 emojis in meme channel only IMO

Reply:
Move the two meme rater actions to a new list in the chain, then have each raise EndProcessing if they action the event.

Reply:
The meme rater actions need to come _before_ the userinfo. Chains are evaluated from the top down.